data/progress_overwrite.yml

```diff
@@ -39,6 +39,10 @@ ports:
 - name: vtk
   status: ⚠️
   tracking_issue: "https://gitlab.kitware.com/vtk/vtk/-/issues/19337"
+- name: itk
+  import_statement: itk
+  help_wanted: ✅
```

Comment:
```suggestion
  status: ✅
```

Reply:
Also, since when does itk feature modules? We need to for the estimated finish

Reply:
We want to have modules, but we don't have a firm plan, mostly due to lack of resources. How about 3 years from now (estimate)?

Reply:
Ah wait so itk does not (yet) support modules? I was confused by the `import_statement: itk` and I figured that it already has modules support. Then my suggestion from earlier does not make sense. Just remove the import_statement. We'll add it when it does support modules